### DISCUSSION TITLE
TLS/SSL for 16 bit

### DISCUSSION THREAD

**Discussion by toncho11** [jbruchon/elks#1615] [Ideas]
It is actually possible to use contemporary TLS/SSL on a 16 bit architecture. 

It is explained here: https://www.dialup.net/wingpt/tls.html

https://www.wolfssl.com can be compiled for 16 bit system. This could even allow web browsing on systems like 486/586 with ELKS.

**Comment by tyama501** [comment]
I saw that recently too.
Also WinGPT for windows 3.1 really cool.
https://www.dialup.net/wingpt/

**Comment by ghaerr** [comment]
I see that WinGPT uses TLS 1.3, which WolfSSL supports. Don't tell me we someone now wants an ELKS-GPT port!! :)
